README.md

```diff
@@ -1,12 +1,30 @@
 # node-rhsplib
 
-This package enables Node.js applications to control devices that speak the REV Hub Serial Protocol (such as the REV Robotics Expansion Hub).
+This project enables Node.js applications to control devices that speak the REV Hub Serial Protocol (such as the REV Robotics Expansion Hub).
+
+## package structure
+
+This project uses [lerna](https://lerna.js.org) for
+multi-package builds.
+
+We have several packages currently:
+
+1. rhsplib: Raw interaction with RHSPLib.
+1. expansion-hub: High-level interaction with a REV expansion hub.
+1. sample: Command line app useful for testing and showing features.
+
+## building and other tasks
+
+To build the app, run `lerna run build`.
+If you have other tasks you wish to create, define 
+them in the package.json like normal, and run 
+`lerna run mytask`
```

Comment:
This is only necessary when you're defining the tasks in the per-package `package.json` files, right? Tasks defined in the top-level `package.json` file can be run using `npm run`.

Reply:
Added a note